src/http.ts

```diff
@@ -99,7 +99,10 @@ export async function getIpFamily(authHost: string): Promise<IpFamily> {
     },
     0,
   );
-  return res.success ? family : undefined;
+
+  const ipv6Incompatible = (<FailedResponse>res).errorCode == ErrorCodes.dnsNotFound;
```

Comment:
could we have that `ErrorCodes.dnsNotFound` for any other reason rathe then piv6 compatibility? if so, is `undefined` return will correct one for such cases?

Reply:
Yes. If ENOTFOUND is thrown, this will be true but will be also an indication that it's incompatible with IPv6.

Since `ErrorCodes.dnsNotFound` is a purely a customised error code that gets assigned only within try/catch, the handling will be the consistent with the current CLI IPv6 incompatibility handling.